test/unittests/skills/test_fallback_skill.py

```diff
@@ -0,0 +1,49 @@
+from unittest import TestCase, mock
+
+from mycroft.skills import FallbackSkill
+
+
+def setup_fallback(fb_class):
+    fb_skill = fb_class()
+    fb_skill.bind(mock.Mock(name='bus'))
+    fb_skill.initialize()
+    return fb_skill
+
+
+class TestFallbackSkill(TestCase):
+    def test_life_cycle(self):
+        """Test startup and shutdown of a fallback skill.
+
+        Ensure that an added handler is removed as part of default shutdown.
+        """
+        self.assertEqual(len(FallbackSkill.fallback_handlers), 0)
+        fb_skill = setup_fallback(SimpleFallback)
+        self.assertEqual(len(FallbackSkill.fallback_handlers), 1)
+        self.assertEqual(FallbackSkill.wrapper_map[0][0],
```

Comment:
Keeping my nitpick hat on, do we also want to check the length of the object here? If somehow multiple tuples were assigned it could still pass.

Reply:
It's probably a good idea to check, but perhaps it's enough to add that check in the lifecycle test so it's tested at exactly one place.

Reply:
I added this check in the lifecycle test